docs/atomic-run.1.md

```diff
@@ -49,6 +49,9 @@ These defaults are suggested values for your container images.
 **SUDO_GID**
   The `SUDO_GID` environment variable.  This is useful with the docker `-u` option for user space tools.  If the environment variable is not available, the default GID of the value for `SUDO_UID` is used.  If this value is not available, the value of `/proc/self/loginuid` is used.
 
+**RUN_OPTS**
```

Comment:
> How about we just call this OPTS_FILE and then we can use it for INSTALL, RUN, UNINSTALL. Any reason to have multiple?

Truth be told, I am able to imagine just the `RUN` use case, when in `INSTALL` you set the file to be consumed in `RUN`. I'm a bit afraid that using one file for all three (`INSTALL`, `RUN`, and `UNINSTALL`) might cause clashes because `RUN` might need some other options than `UNINSTALL`.

Do you have some examples of uses of this mechanism for `INSTALL` and `UNINSTALL`?

Or do you envision the `LABEL`s being different for each phase, just the "variable" where it is exposed being the same (`OPTS_FILE`)?